contentcuration/contentcuration/frontend/channelEdit/views/files/__tests__/fileUpload.spec.js

```diff
@@ -71,9 +71,9 @@ describe('fileUpload', () => {
   });
   describe('computed', () => {
     it('should map the files to the correct presets', () => {
-      expect(wrapper.vm.primaryFileMapping[0].file.id).toBe('file-3');
-      expect(wrapper.vm.primaryFileMapping[1].file.id).toBe('file-1');
-      expect(wrapper.vm.primaryFileMapping).toHaveLength(2);
+      expect(wrapper.vm.primaryFileMapping[1].file.id).toBe('file-3');
+      expect(wrapper.vm.primaryFileMapping[2].file.id).toBe('file-1');
```

Comment:
Were these failing before or just not testing what they were supposed to test?

Reply:
They were just brittle to the order of file presets - because the bloompub preset happened to go before epub and pdf, (because alphabetical) it moved them each down a place.